flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -2189,6 +2267,8 @@ SqlCreate SqlCreateExtended(Span s, boolean replace) :
         |
         create = SqlCreateTable(s, replace, isTemporary)
         |
+        create = SqlCreateMaterializedTable(s)
```

Comment:
What is the behavior when create temporary materialized table?

Reply:
Updated, an unsupported exception will be thrown when temporary materialized appears.